src/lib/contents/docs/menu.ts

```diff
@@ -3,145 +3,248 @@ import type { MenuStatus, MenuEntry } from "$lib/types/menu-entry.type";
+      M("Requirements", "configure/self-hosted/latest/requirements"),
+      M("Telemetry", "configure/self-hosted/latest/telemetry"),
+      M("Release Policies", "configure/self-hosted/latest/releases"),
+      M("Upgrade Guides", "configure/self-hosted/latest/upgrade-guides"),
     ]),
   ]),
-  M("References", "references", [
-    M("Compatibility Matrix", "references/product-compatibility-matrix?user"),
-    M(".gitpod.yml", "references/gitpod-yml"),
-    M("Command Line Interface", "command-line-interface"),
-    // M("Custom Docker image", "references/gitpod-dockerfile"),
-    // M("Architecture", "references/architecture"),
-    M("Roadmap", "references/roadmap"),
-    M("Gitpod Releases", "references/gitpod-releases"),
+  M("Billing", "configure/billing", false, [
+    // M("Subscriptions", "configure/billing/subscriptions"),
+    M("Personal plans", "configure/billing/personal-plans"),
+    M("Team plans", "configure/billing/team-plans"),
+    M("Team billing", "configure/billing/team-billing", false, [], "beta"),
+    M(
+      "Usage-based billing",
+      "configure/billing/usage-based-billing",
+      false,
+      [],
+      "Early Access"
+    ),
+  ]),
+
+  M("References", "references", true),
+  M(".gitpod.yml", "references/gitpod-yml"),
+  M("IDEs & editors", "references/ides-and-editors", false, [
+    M("VS Code Browser", "references/ides-and-editors/vscode-browser"),
+    M(
+      "VS Code Desktop",
+      "references/ides-and-editors/vscode",
+      false,
+      [],
+      "beta"
+    ),
+    M(
+      "IntelliJ IDEA",
+      "references/ides-and-editors/intellij",
+      false,
+      [],
+      "beta"
+    ),
+    M("GoLand", "references/ides-and-editors/goland", false, [], "beta"),
+    M("PhpStorm", "references/ides-and-editors/phpstorm", false, [], "beta"),
+    M("PyCharm", "references/ides-and-editors/pycharm", false, [], "beta"),
+    M("CLion", "references/ides-and-editors/clion", false, [], "soon"),
+    M("Rider", "references/ides-and-editors/rider", false, [], "soon"),
+    M("RubyMine", "references/ides-and-editors/rubymine", false, [], "soon"),
+    M("WebStorm", "references/ides-and-editors/webstorm", false, [], "soon"),
+    M(
+      "Local Companion",
+      "references/ides-and-editors/local-companion",
+      false,
+      [],
+      "beta"
+    ),
+    M("VS Code extensions", "references/ides-and-editors/vscode-extensions"),
+    M("VS Code settings sync", "references/ides-and-editors/settings-sync"),
+    M(
+      "Command Line (SSH)",
+      "references/ides-and-editors/command-line",
+      false,
+      []
+    ),
+    M("FAQs", "references/ides-and-editors/faqs", false, []),
+  ]),
+  M("Integrations", "integrations", false, [
+    M("Tailscale", "integrations/tailscale"),
+    M("JetBrains Gateway", "integrations/jetbrains-gateway", false, [], "beta"),
+  ]),
+  M("Gitpod CLI", "references/gitpod-cli"),
+  M("Compatibility", "references/compatibility"),
```

Comment:
Any reason we are not referring to the user view here anymore?

```suggestion
  M("Compatibility", "references/compatibility?user"),
```